scripts/glutton/update/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[workspace]
```

Comment:
Why should this not be in the main workspace?  
This will make it harder for the monorepo and when updating dependencies.

Reply:
I thought about that, but on the other hand I didn't want to increase Cumulus compile time, which is already high. After Gluttons Parachains fulfil their mission, I do not think we'll need this script anymore. What do you think?

Reply:
It does not increase compile time, since this is not part of the cumulus binary. I hope most devs use something like `cargo c -p polkadot-parachain`.

Reply:
Ok, I moved it to the main workspace